pegasus/sites.v3/code.org/public/educate/professional-learning/middle-high.md.erb

```diff
@@ -10,8 +10,27 @@ theme: responsive
   details summary {
     cursor: pointer;
   }
+
+  a.tertiary-link {
+    color: #00adbc;
+    font-size: 14px;
+    font-family: "Gotham 4r", sans-serif;
+    font-weight: bold;
+    text-decoration: none;
+  }
+    a.tertiary-link .fa {
+      font-size: 12px;
+      font-weight: bold;
+      margin-left: 8px;
+    }
+
+  .optimizely-variant-a {
+    display: none;
+  }
```

Comment:
This ensures the variant isn't visible when Optimizely isn't interfering.  We override this in Optimizely by applying `display: block;` to individual elements, which are always `div`s in our case.

Reply:
where is the config for what optimizely is applying `display: block;` to?

Reply:
Shown in the PR description, copied here:
![image](https://user-images.githubusercontent.com/1615761/68983106-59ccdc00-07be-11ea-89dd-dbbbf8936eff.png)
I think [this](https://app.optimizely.com/v2/projects/12977480133/experiments/16866320542/variations/16838333854) is a direct link to that config.